java/pom.xml

```diff
@@ -47,7 +46,7 @@
         <dependency>
             <groupId>sh.platform</groupId>
             <artifactId>config</artifactId>
-            <version>2.2.0</version>
+            <version>2.2.1-SNAPSHOT</version>
```

Comment:
We should be targeting a released version, no?  It's not a reliable example otherwise.

Reply:
Oh!  It's because of the new services, I see.  I assume that will get tagged first, then this updated before merge?

Reply:
Yeap, we need to merge this one first: https://github.com/platformsh/config-reader-java/pull/8